src/lib/features/project/project-service.test.ts

```diff
@@ -24,12 +35,17 @@ describe('enterprise extension: enable change requests', () => {
             TEST_AUDIT_USER,
         );
 
-        return service;
+        return {
```

Comment:
this makes code testable without breaking service encapsulation and without ts-expect-error